### ISSUE TITLE
KFold Logic

### ISSUE THREAD

**Issue by kocemir** [TencentAILabHealthcare/scBERT#58]
Hello, I want to clarify my understanding regarding the code below. You are splitting data 5 times but you are using the final split scheme in the training. I think, you should use this splitting in a global for loop that also takes inside the training and validation loops of the model. Am I wrong here?
""""
**sss = StratifiedShuffleSplit(n_splits=1, test_size=0.2, random_state=SEED)
for index_train, index_val in sss.split(data, label):
    data_train, label_train = data[index_train], label[index_train]
    data_val, label_val = data[index_val], label[index_val]
    train_dataset = SCDataset(data_train, label_train)
    val_dataset = SCDataset(data_val, label_val)

train_loader = DataLoader(train_dataset, batch_size=BATCH_SIZE, shuffle=True)
val_loader = DataLoader(val_dataset, batch_size=BATCH_SIZE, shuffle=False)**

""""

I
